### DISCUSSION TITLE
Issue while Creating Workload Item

### DISCUSSION THREAD

**Discussion by SanchanaMohankumar** [microsoft/Microsoft-Fabric-workload-development-sample#28] [Q&A]
On creating a workload item, the workload item creation fails and throws the following exception, as shown in Image 1. I have also attached the request ID, 9877c1f5-55f9-0351-a63e-98745bd57243, for reference. This is a transient issue. Please do provide your input on the following issue.

Image1 
<img width="164" alt="Exception" src="https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/assets/175164325/2a654411-a939-4bb5-ac44-c20c9786bb8d">

Request ID:
9877c1f5-55f9-0351-a63e-98745bd57243

**Comment by anitamayorov** [comment]
Hi @SanchanaMohankumar,

Can you elaborate on your scenario? Is this a development scenario with local BE running? 
Were you able to connect the workload using DevGateway and you have a running workload while trying to create the item?

**Reply by SanchanaMohankumar** [reply]
Hi @anitamayorov,
This is a production scenario that we have deployed. If you need any more information, please let me know, and I will be happy to provide additional details.

**Reply by anitamayorov** [reply]
Hi @SanchanaMohankumar,

Can you please verify that your DevGateway is not running on the same capacity you are trying to create the item on? Perhaps on another machine. If so, it is trying to access your dev workload.
